rmw_fastrtps_cpp/src/type_support_common.cpp

```diff
@@ -119,6 +119,11 @@ bool TypeSupport::deserializeROSmessage(
       "Fast CDR exception deserializing message of type %s.",
       getName());
     return false;
+  } catch (std::bad_alloc &) {
```

Comment:
```suggestion
  } catch (const std::bad_alloc &) {
```

(it probably makes no difference in performance here, but it is good practice in my opinion)

Reply:
Sure. Done in 2e8c1e6ef959429b17412c7753b3bfbcf4936489